src/platform/site-wide/sass/merger.scss

```diff
@@ -557,9 +559,17 @@
             margin: 2px;
           }
 
+          &:focus:hover {
+            background-color: $mm-gray;
+            font-weight: bold;
+            margin: 2px;
+            width: 98%;
+          }
+
           &:hover {
             background-color: $mm-gray;
             font-weight: bold;
+            width: 100%;
```

Comment:
Would it make sense to apply `width: 100%` directly to the parent instead of only hover? Or would that break things?

Reply:
Yeah, I think we might want to change this. It creates a jumpiness when you hover over and off again, because the you can see the right margin appear as the grey background fades.

Reply:
@ncksllvn Let mw try it out and see what happens. I know I was having issues with this.